Inspicio/Migrations/ApplicationDbContextModelSnapshot.cs

```diff
@@ -107,9 +107,11 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 
                     b.Property<string>("Description");
 
-                    b.Property<int>("NoOfDislikes");
+                    b.Property<int>("NoOfApprovals");
```

Comment:
The `Review` object is storing the vote (approve/reject/needs work) and, so you can do something like `var approvedCount = _context.Review.Count(x => x.ImageId == data.ImageID && x.Approved);` - which would let you simplify the `ChangeRating` method a bit (especially if the bools are changed to an enum).